client/app/_services/testsruns.service.js

```diff
@@ -129,6 +130,15 @@
             return isSearchActive() && !_searchParams.hasOwnProperty('query');
         }
 
+        function isModalSearchActive() {
+            if(Object.keys(_searchParams).length > Object.keys(DEFAULT_SC).length + 1 || isOnlyAdditionalSearchActive()) {
+                return true;
+            }
+            else {
```

Comment:
1 - else state should be in 1 line
2 - you don't need to wrap you returning in else state is this case